### PR TITLE
[INTER-549] Revisit

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -24,10 +29,10 @@ jobs:
...
 
       - name: Build apps
-        run: yarn build
+        run: pnpm build
 
       - name: Release
-        run: yarn release
+        run: pnpm release
         env:
           GITHUB_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
```
Comment (ilfa) [inline .github/workflows/release.yml:37]: Which token do we use here?

File: package.json [hunk 2]
```diff
@@ -6,67 +6,71 @@
...
-  "workspaces": [
-    "packages/**"
-  ],
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@commitlint/cli": "^17.6.3",
-    "@commitlint/config-conventional": "^17.6.3",
-    "@semantic-release/changelog": "^6.0.1",
-    "@semantic-release/git": "^10.0.1",
```
Comment (ilfa) [inline package.json:32]: You removed semantic release plugins from deps, but release process still the same as I understand.

File: packages/chrome-extension-e2e/package.json [hunk 3]
```diff
@@ -4,12 +4,6 @@
   "private": true,
   "description": "Stores e2e tests for chrome extension;",
   "scripts": {
-    "e2e": "npx playwright test --headed"
-  },
-  "devDependencies": {
-    "@playwright/test": "latest",
```
Comment (ilfa) [inline packages/chrome-extension-e2e/package.json:10]: Why did you remove dev-deps?

Comment (TheUnderScorer) [reply packages/chrome-extension-e2e/package.json:10]: I moved them to root `package.json`

**Review by ilfa — APPROVED** [review]
